<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>placeu</class>
 <widget class="QWidget" name="placeu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1019</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Места занятий</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>70</y>
     <width>448</width>
     <height>441</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QTextEdit" name="tePlaceU">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>431</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="twPlaceU">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>12</y>
      <width>431</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBackPlU">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>330</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Назад</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bntConnect">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>330</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Подключение</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
